website/docs/intro-ee.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Enterprise Edition
+hide_title: true
+---
+import TierLabel from "./_components/TierLabel";
+import Link from "@docusaurus/Link";
+
+# Weave GitOps Enterprise Edition <TierLabel tiers="Enterprise" />
+
+:::tip Ready for more GitOps?
+To purchase entitlement to Weave GitOps Enterprise Edition please contact [sales@weave.works](mailto:sales@weave.works)
+:::
+
+WGEE provides ops teams with an easy way to assess the
+health of multiple clusters in a single place. It shows cluster information such as
+Kubernetes version and number of nodes and provides details about the GitOps operations
+on those clusters, such as Git repositories and recent commits. Additionally, it
+aggregates Prometheus alerts to assist with troubleshooting.
+
+If you have already purchased your entitlement, head to the [installation page](./installation/weave-gitops-enterprise/index.mdx).
+
+## Feature breakdown
```

Comment:
Considering this list: https://docs.google.com/presentation/d/1D5o7dyLCKg_riO_GkfLPLu2jQiUvNbLIGQGFbtJndAk/edit#slide=id.g1eadeea455d_1_5 + GitOpsSets

Reply:
I will add gitops sets once @foot 's docs pr is merged.

As for the other list, it deviates a bit from what we have on the corp site (which is where i lifted this one from). It feels more sensible (and less confusing) to be consistent in the stuff we advertise (using the same keywords and language). If we want to use this list, we would need to change it in corp too.

Reply:
Thats a good shout. Maybe @joe-terrible can advise and help to compile a 'definitive' list.

Reply:
👍 am i good to merge this as-is then we can have a later ticket to do list changes once we have the definitive one?